v1/protocol_binding/http_binder.cc

```diff
@@ -0,0 +1,122 @@
+#include "http_binder.h"
+
+namespace cloudevents {
+namespace binding {
+
+using ::io::cloudevents::v1::CloudEvent;
+using ::io::cloudevents::v1::CloudEvent_CloudEventAttribute;
+using ::cloudevents::format::StructuredCloudEvent;
+using ::cloudevents::format::Format;
+using ::cloudevents::format::Formatter;
+using ::cloudevents::formatter_util::FormatterUtil;
+using ::cloudevents::cloudevents_util::CloudEventsUtil;
+
+typedef boost::beast::http::request<boost::beast::http::string_body> HttpRequest;
+typedef absl::flat_hash_map<std::string, CloudEvent_CloudEventAttribute> CeAttrMap;
+
+constexpr absl::string_view kHttpContentKey = "Content-Type";
+
+template <>
+absl::StatusOr<std::string> Binder<HttpRequest>::GetContentType(HttpRequest& http_req) {
+    auto iter = http_req.base().find(kHttpContentKey.data());
+    if (iter == http_req.base().end()) {
+        return std::string("");
```

Comment:
would `return "";` work?

Reply:
I get a conversion error, I think with absl::StatusOr I always need explicit type declaration if it's implicit/ polymorphic

```
error: could not convert '""' const char [1]' to 'absl::StatusOr<std::__cxx11::basic_string<char> 
```

Reply:
I see. Thanks.